### DISCUSSION TITLE
Filament Production perfomance issue

### DISCUSSION THREAD

**Discussion by ExsyDev** [filamentphp/filament#14376] [Help]
### Package

Panel builder

### Package Version

3.0.0

### How can we help you?

Hello everyone. 

Locally I have dockerized Laravel Filament Application, with perfomance no issues, because my machine is: Macbook Pro 13 M1 cpu 16gb and 500gb SSD disk.
When I publish my dockerized application to linux server with 2vcpu core and 4gb RAM, I get slow loading from server response time (on resources index page), if I go to show resource page I do not see these issue.

What I can do?

On production:

I have nginx proxy server to my dockerized image with application. In docker I have nginx + phpfpm and redis server.
On production I run this commands:

composer install --optimize-autoloader --no-dev
php artisan optimize
php artisan filament:optimize

Laravel DebugBar is false on production.
With Database Query responses I have time execution max: 60ms

**Comment by MohsenSouri** [comment]
We have the same issue does anybody have a solution about this slow loading on the index pages?
